riscv.c

```diff
@@ -597,20 +642,20 @@ static bool op_system(struct riscv_t *rv, uint32_t inst)
             rv->io.on_ecall(rv);
             break;
         case 1:  // EBREAK
-            rv->io.on_ebreak(rv);
+            rv_breakpoint(rv,rv->PC);
```

Comment:
Did you indent the source files with `clang-format -i`?

Reply:
By the way, you should explain the reason and consideration why you would like to perform such modification.